src/rest-api/config.ts

```diff
@@ -0,0 +1,5 @@
+
+// Default values
+export const config = {
+  useRealAPI: (process.env.REST_SRC === 'real'),
```

Comment:
Nobody invited me here, but I would keep the word `real` as Braulio used... even better: I would be using the word `mocked` because with that word there's no doubt what we are referring to. 
So the config variable would be named as: `useRealAPI` or `useMockedAPI`. since these both the **mocked** and the **real** are REST APIs...